src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7900,7 +7900,7 @@ def test_aks_migrate_cluster_to_cilium_dataplane(self, resource_group, resource_
         ])
 
         # update to enable cilium dataplane
-        update_cmd = 'aks update -g {resource_group} -n {name} --network-dataplane=cilium'
+        update_cmd = 'aks update -g {resource_group} -n {name} --network-dataplane=cilium --network-policy=cilium'
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=81175429&view=results) to validate the change.

Reply:
Please add check for this new added param.

Reply:
See the comment below:
We want to avoid referencing 'none' (that was introduced in API version 2023-08-02-preview) until it is ready. See comment: https://github.com/Azure/azure-cli-extensions/pull/6809#discussion_r1337806199

Reply:
Is the live test passed? Please upload the recoding file.

Reply:
uploaded the recording file and attached to the description